### PR TITLE
Adding search functionality

### PR DESCRIPTION
**Pull request by Will-Cooper**
separate page for query coordinate and add main searchbar to homepage

### CODE REVIEW

**Comment by Will-Cooper** [comment]
Happy that that test failed with the change in astrodbkit2, exactly as intended!
Should be okay now, and looks okay on the website

**Comment by dr-rodriguez** [comment]
Tried `03:42:31.8  12:16:22` (and a few others) and they fail because I put 2 spaces between the ra/dec. I think the error message is clear enough, but could we perhaps strip the input string of any duplicate spaces?

**Comment by dr-rodriguez** [comment]
May also be good to also split by tabs since people copying from a table may have ra/dec delimited by tabs

**Comment by Will-Cooper** [comment]
Made that change to the string parsing and added in optional radius + minor html/ jquery enhancements @dr-rodriguez

File: simple_app/utils.py [hunk 1]
```diff
@@ -630,6 +630,43 @@ def multidfquery(results: Dict[str, pd.DataFrame]) -> Dict[str, Optional[str]]:
...
+        First output
+    b
+        Second output
+    c
+        Third output (optional)
+    """
+    try:
+        qrysplit: np.ndarray = np.array(s.replace('\t', ' ').lower().strip().split(' '))
+        qrysplit = qrysplit[np.logical_not(qrysplit == '')]
+        qrylen = len(qrysplit)
```
Comment (dr-rodriguez) [inline simple_app/utils.py:654]: There may be ways to write this as a regex, but for now this works

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**